store/sqlstore/channel_store.go

```diff
@@ -3901,19 +3901,17 @@ func (s SqlChannelStore) UpdateSidebarCategories(userId, teamId string, categori
 			}
 
 			// And then add the new ones
-			var preferences []interface{}
-
 			for _, channelID := range category.Channels {
```

Comment:
This is actually roughly how gorp's bulk insert method works internally. I was originally hesitant to switch because I thought they'd be doing something special for a bulk insert, but it's also just looping an insert statement. The only thing that's really different is that `SqlPreferenceStore.save` does a select followed by an update/insert on Postgres since we don't yet require only versions with a proper upsert.